### PR TITLE
World tour

### PR DESCRIPTION
**Pull request by insectengine**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/master/docs/src/main/asciidoc **

### CODE REVIEW

**Comment by maxandersen** [comment]
Title says world tour but all content seem to be minor fixes to guides/books - wrong files?

File: blog.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: blog
-title: "Quarkus - Blog"
+title: "Blog"
```
Comment (gsmet) [inline blog.md:3]: I'm unsure this is a good change given we won't have `Quarkus` in the `<title>` anymore.

**Review by gsmet — CHANGES_REQUESTED** [review]

File: _sass/layouts/guides.scss [hunk 2]
```diff
@@ -277,8 +277,10 @@ div.guides-configuration-reference {
 }
 
 select#guides-version-dropdown {
-  color: black;
-  font-size: 1.7rem;
+  background-color: transparent;
+  color: $white;
+  border: 1px solid $white;
+  font-size: 1.35rem;
```
Comment (gsmet) [inline _sass/layouts/guides.scss:283]: This change doesn't work very well for Guide pages: http://localhost:4000/guides/getting-started-reactive .

File: _guides/guides.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
 layout: guides-index
-title: Quarkus - Guides
+title: Guides - Latest
```
Comment (gsmet) [inline _guides/guides.md:3]: I wonder if we should have this done dynamically in the template and maybe with a badge of some sort (we could do it for all the versions in one go)?

I can do the code if you can just do the layout part with a badge for the version.

**Review by gsmet — CHANGES_REQUESTED** [review]
I added some more comments.

File: _sass/layouts/guides.scss (hunk 2 above)
Comment (insectengine) [reply _sass/layouts/guides.scss]: > This change doesn't work very well for Guide pages: http://localhost:4000/guides/getting-started-reactive .

I've committed more changes do address the two background color situations.

**Comment by gsmet** [comment]
I rebased and will merge as soon as CI is green.

**Review by gsmet — APPROVED** [review]

**Pull request merged**